styles.css

```diff
@@ -0,0 +1,1115 @@
+}
+
+.lang-font {
+  font-family: Poppins, sans-serif;
+  font-weight: bold;
+  color: #344563;
+}
+
+.lang {
+  margin: 30px;
+  margin-top: 50px;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  padding-right: 40px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.lang-margin {
+  margin-top: 4px;
+}
+
+.lang-2 {
+  margin: 20px;
+  padding-top: 6px;
+  padding-bottom: 25px;
+  padding-right: 40px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.lang3 {
+  border-bottom: 1px solid rgb(202, 200, 200);
+}
+
+.icons {
+  height: 15px;
+  width: 15px;
+}
+
+.language {
+  display: flex;
+  margin-left: 10px;
+  margin-bottom: 10px;
+  background-color: #f6f6f9;
+  border-radius: 8px;
+  padding: 10px;
+}
+
+.lang-text {
+  font-family: Poppins, sans-serif;
+  color: #344563;
+  padding-top: 10px;
+  padding-left: 15px;
+  font-style: normal;
+}
+
+.contact-section {
+  background: #fff;
+}
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+  background: #6070ff;
+  background-image: url(./images/contact-shapesm.svg);
+  background-repeat: no-repeat;
+  background-position: right;
+  border: 1px solid white;
+  border-top-left-radius: 110px;
+  padding-bottom: 48px;
+}
+
+.con-title {
+  color: #fcfcfc;
+  justify-content: center;
+  font-weight: bold;
+  line-height: 40px;
+  font-size: 40px;
+  font-family: Poppins, sans-serif;
+  margin: 150px auto 12px;
+}
+
+.con-paragraph {
+  font-family: Poppins, sans-serif;
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+  justify-content: center;
+  text-align: center;
+  font-weight: normal;
+  margin: 0 auto;
+  padding: 10px 50px;
+}
+
+.con-form {
+  display: flex;
+  flex-direction: column;
+  margin: 30px;
+  gap: 20px;
+}
+
+.form-text {
+  border: 1px solid white;
+  border-radius: 8px;
+  height: 48px;
+  padding: 15px;
+  font-family: Poppins, sans-serif;
+}
+
+.error-message {
+  color: red !important;
+  margin-top: 0.2rem;
+  font-size: 14px;
+  text-transform: uppercase;
+}
+
+.con-but {
+  justify-content: start;
+  width: 140px;
+  height: 48px;
+  padding: 10px;
+  border: 1px solid white;
+  border-radius: 8px;
+  font-family: Poppins, sans-serif;
+  font-size: 17px;
+  font-weight: 500;
+  color: #2e63e7;
+}
+
+.message {
+  height: 170px;
+  border-radius: 8px;
+}
+
+.skills-section {
+  position: relative;
+  bottom: 30%;
+}
+
+.technologies-langu li {
+  background-color: #ebebff;
+  padding: 10px;
+  border-radius: 10px;
+  font-weight: 500;
+  font-family: poppins, sans-serif;
+  justify-content: space-between;
+  margin-right: 20px;
+  display: inline-block;
+}
+
+.menu-desktop {
+  display: none;
+}
+
+.desktop-image {
+  display: none;
+}
+
+.desktop_show,
+.optiond2_full,
+.instruction-show,
+.dcanopy2_face,
+.dcanopy2_facebook,
+.desktop-image {
+  display: none;
+}
+
+@media (min-width: 992px) {
+  .sidebar {
+    display: none;
+  }
```

Comment:
- [ ] Great job implementing the styles with a mobile first approach, however it's specified that we should use `min-width: 768px`, kindly change that.

Reply:
thanks I have made the corrections